server/py/services/api/api/api.py

```diff
@@ -189,6 +190,11 @@
     tags=["nuclio"],
     dependencies=[Depends(deps.authenticate_request)],
 )
+api_router.include_router(
+    alert_activations.router,
+    tags=["alert-activations"],
+    dependencies=[Depends(deps.authenticate_request)],
```

Comment:
@alonmr more for you - why do we need to pass `dependencies=[Depends(deps.authenticate_request)]` for every `include_router` if it's the same for all?